plugins/a11y-text-wand/index.js

```diff
@@ -22,11 +22,11 @@ class A11yTextWand extends Plugin {
         this.summary(" ");
         this.panel.render();
 
-        $(document).on("mousemove.wand", function(e) {
-            let element = document.elementFromPoint(e.clientX, e.clientY);
+        $(document).on("mousemove.wand", function(e, testElement) {
```

Comment:
The `testElement` here was necessary to be able to test the a11y-text-wand because `document.elementFromPoint()` isn't available in the test environment.

Reply:
Thank you for these extra comments. Really helps review the code. :)

Reply:
Yes, that makes sense since `jsdom` isn't a browser.